common/data_refinery_common/test_job_management.py

```diff
@@ -1,18 +1,16 @@
-import os
-import psutil
 
 from django.test import TestCase, tag
 from typing import List
 from unittest.mock import patch, call
 from urllib.error import URLError
 
-from data_refinery_workers.downloaders import utils
+from data_refinery_common.job_management import create_processor_job_for_original_files
 
 
 class UtilsTestCase(TestCase):
-    @tag('downloaders')
+    @tag('processor')
```

Comment:
This tag is important for where in the test workflow we run the test. We will filter tests for the tag `downloaders`, however we don't for `processor` because we have processor-specific tags for each of the processor images. Basically this tag is specifying which image should be used to process it, and there's no `processor` image. Honestly this can probably still be run in the `downloaders` tests because it doesn't use code that's specific to any processor image.

Reply:
Alternatively the tag could probably just be removed and this could run in the common tests.

Reply:
I like that better, I'm gonna remove it!